assets/pages/support-landing/supportLanding.scss

```diff
@@ -17,14 +17,14 @@
 @import '../../components/pageSection/pageSection';
 @import '../../components/doubleHeading/doubleHeading';
 @import '../../components/secure/secure';
-@import '../../containerisableComponents/contributionAwarePaymentLogos/contributionAwarePaymentLogos';
-@import '../../containerisableComponents/contributionSelection/contributionSelection';
+@import '../../components/contributionAwarePaymentLogos/contributionAwarePaymentLogos';
```

Comment:
Another place that we could replace `../..` with `~` to have a consistent style.